base/models/models.go

```diff
@@ -51,7 +52,7 @@ type SystemAdvisories struct {
 	ID            int
 	SystemID      int
 	AdvisoryID    int
-	FirstReported time.Time
+	FirstReported *time.Time
```

Comment:
This field corresponds to `first_reported` column in db which is NOT NULL, so it doesn't have to be pointer.

Reply:
If it isn't null its set to `0000-00- .....` By gorm. This is wrong. The field is automatically set to current timestamp by a database trigger if null is provided.